chroma-manager/tests/utils/remote_firewall_control.py

```diff
@@ -187,19 +187,28 @@ def process_rules(self):
         result = self.remote_access_func(self.address, self.firewall_list_cmd)
 
         if result.rc != 0:
-            from chroma_common.lib.shell import Shell
-            raise RuntimeError('''process_rules(): remote shell command failed unexpectedly (%s), is firewall-cmd running? (%s) (%s)
-systemctl status firewalld:
-%s
-
-systemctl status polkit:
-%s
-
-journalctl -n 100:
-%s''' % (result.rc, result.stdout, result.stderr,
-         Shell.run(['systemctl', 'status', 'firewalld']).stdout,
-         Shell.run(['systemctl', 'status', 'polkit']).stdout,
-         Shell.run(['journalctl', '-n', '100']).stdout))
+            if result.rc == 254 and ' is not registered' in result.stderr:
+                from time import sleep
+                # restart polkit to attempt to address https://bugzilla.redhat.com/show_bug.cgi?id=1221483
+                self.remote_access_func(self.address, 'systemctl restart polkit')
```

Comment:
Ahhh.  OK.  I thought maybe you dug up some additional information that was not in the RH ticket that suggested a restart would fix it.  I guess we will find out.  But in the meanwhile we really should see if EL7 needs an updated firewalld and get that in motion if it does.

In the least we should at least land the fixes to the discovery.